huaweicloud/resource_huaweicloud_nat_dnat_rule_v2.go

```diff
@@ -148,17 +170,17 @@ func resourceNatDnatRuleCreate(d *schema.ResourceData, meta interface{}) error {
 		params["floating_ip_id"] = floatingIPIDProp
 	}
 
-	internalServicePortProp, err := navigateValue(opts, []string{"internal_service_port"}, nil)
+	internalServicePortRangeProp, err := navigateValue(opts, []string{"internal_service_port_range"}, nil)
```

Comment:
we should add `internal_service_port_range` into params instead of replacing `internal_service_port`